source/clear-linux/tutorials/tutorials.rst

```diff
@@ -28,3 +28,4 @@ Explore our tutorials to discover what you can do with |CL|!
    yubikey-u2f
    nvidia
    dars
+   gce/gce
```

Comment:
This document is probably better off in the getting started instead of tutorials. I imagine TCS will advise.

Reply:
I put this document in the same hierarchy as the documents for Azure and AWS. I'd leave the decision to TCS. :)